project/src/main/webapp/style.css

```diff
@@ -1,9 +1,251 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/*** General ***/
+body {
+    margin: 0px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.app-header {
```

Comment:
Just FYI, if the CSS you're doing applies to just one element and isn't reused across multiple elements, it's generally better to use `id` instead of `class`. Then, in the CSS, you use `#app-header` (for example) instead of `.app-header`.